awscli/customizations/emr/emrutils.py

```diff
@@ -248,13 +240,13 @@ def call_and_display_response(session, operation_name, parameters,
                               parsed_globals):
         cli_operation_caller = CLIOperationCaller(session)
         cli_operation_caller.invoke(
-            session.get_service('emr').get_operation(operation_name),
+            'emr', xform_name(operation_name),
```

Comment:
Good catch. ``CLIOperationCaller`` might have changed from when I made this change.